src/frontend/src/lib/components/ui/Logo.svelte

```diff
@@ -5,9 +5,16 @@
 
 	export let src: string | undefined;
 	export let alt = '';
-	export let size: string;
+	export let size: 'small' | 'medium' | 'big' = 'medium';
```

Comment:
if it's mostly `small`, shouldn't the default be `small`?

Reply:
> Done, please check last commit. It is mostly used as `'small'` , so maybe we can default to that?

indeed, see my comment above, i thought the same, agreed